tests/check_public_interfaces.py

```diff
@@ -203,9 +203,6 @@ def test_gpg_cmds(self):
     with self.assertRaises(securesystemslib.exceptions.UnsupportedLibraryError):
       securesystemslib.gpg.functions.create_signature('bar')
 
-    with self.assertRaises(securesystemslib.exceptions.UnsupportedLibraryError):
```

Comment:
Would it make sense to say that we do _not_ raise this exception (regression test)?

Reply:
AFAICT this would require running the `securesystemslib.gpg.functions.verify_signature(None, 'f00', 'bar')` function with correctly formatted values. I'm not really sure how that could be added but this is a great idea.